tests/components/plugwise/test_select.py

```diff
@@ -86,3 +89,20 @@ async def test_legacy_anna_select_entities(
 ) -> None:
     """Test not creating a select-entity for a legacy Anna without a thermostat-schedule."""
     assert not hass.states.get("select.anna_thermostat_schedule")
+
+
+async def test_adam_select_unavailable_regulation_mode(
+    hass: HomeAssistant, mock_smile_anna: MagicMock, init_integration: MockConfigEntry
+) -> None:
+    """Test a regulation_mode non-available preset."""
+
+    with pytest.raises(ServiceValidationError):
+        await hass.services.async_call(
+            SELECT_DOMAIN,
+            SERVICE_SELECT_OPTION,
+            {
+                "entity_id": "select.anna_thermostat_schedule",
+                "option": "freezing",
```

Comment:
can we use constants for entity_id and option?

Reply:
'when in Rome' ... (so a bit more than just these constants